cmd/minishift/cmd/start.go

```diff
@@ -165,6 +165,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	proxyConfig := handleProxies()
 
+	fmt.Println(fmt.Sprintf("-- Starting profile '%s'", minishiftConfig.AllInstancesConfig.ActiveProfile))
```

Comment:
You need to use `constants.ProfileName` instead of `minishiftConfig.AllInstancesConfig.ActiveProfile` as user can start the profile with the `--profile flag` e.g. `minishift --profile foo start`

Reply:
There will be use cases where active profile is `bar` but user is trying to start profile `foo`.

Reply:
Nice! I'm gonna change that